sklearn/model_selection/_split.py

```diff
@@ -1706,12 +1706,19 @@ def _validate_shuffle_split(n_samples, test_size, train_size):
 class PredefinedSplit(BaseCrossValidator):
     """Predefined split cross-validator
 
-    Splits the data into training/test set folds according to a predefined
-    scheme. Each sample can be assigned to at most one test set fold, as
-    specified by the user through the ``test_fold`` parameter.
+    Provides train/test indices to split data into train/test sets using a
```

Comment:
I think "provides" is a poor word choice here given that in this case the user provides the indices

Reply:
It's subtle, but the user doesn't provide the indices that make up each test/train fold. Rather, the user provides what I want to call a "fold assignment" that is converted to said indices. So, instead of "provides" I could put "converts" or "computes"... I'm open to suggestions as I previously decided to use "provides" because it models the docstring for KFold.